tests/generic_config_updater/gu_utils.py

```diff
@@ -305,18 +305,21 @@ def get_asic_name(duthost):
     asic_mapping = gcu_conf["helper_data"]["rdma_config_update_validator"]
     if asic_type == 'cisco-8000':
         asic = "cisco-8000"
-    elif asic_type == 'mellanox' or asic_type == 'vs' or asic_type == 'broadcom':
-        hwsku = duthost.shell(GET_HWSKU_CMD)['stdout'].rstrip('\n')
-        if asic_type == 'mellanox' or asic_type == 'vs':
-            spc1_hwskus = asic_mapping["mellanox_asics"]["spc1"]
-            if hwsku.lower() in [spc1_hwsku.lower() for spc1_hwsku in spc1_hwskus]:
-                asic = "spc1"
-                return asic
-        if asic_type == 'broadcom' or asic_type == 'vs':
+    elif asic_type == 'vs':
+        asic = "vs"
```

Comment:
VS asic can be either broadcom or mellanox, which we would use to check against the conf file, so I suggest we keep vs check with mellanox or broadcom as before

Reply:
But if we check vs with Mellanox or Broadcom, the return value will always be `known`. Is that expected?

Reply:
VS with emulate Mellanox or Broadcom, so return value should not be unknown.
```
admin@vlab-01:~$ sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku
Force10-S6000
```

Reply:
Thanks for checking. The code has been updated.